contributions/catalog/webapps/static/css/style.css

```diff
@@ -527,3 +491,50 @@ input.datadel-api {
 .hidetext { -webkit-text-security: disc; }
 .showtext { -webkit-text-security: none; }
 
+h1 {
```

Comment:
H1 and h4 styles are already added in style.css from line 360. Please remove the duplicates. Thanks.

Reply:
updated

Reply:
Thanks.